app/services/idv/steps/inherited_proofing/verify_wait_step_show.rb

```diff
@@ -0,0 +1,75 @@
+module Idv
+  module Steps
+    module InheritedProofing
+      class VerifyWaitStepShow < VerifyBaseStep
+        include UserPiiManagable
```

Comment:
Going googly-eyed, we actually using this?

Reply:
We are not. I missed some in clean up. Thanks for catching them!